RELEASING.md

```diff
@@ -0,0 +1,15 @@
+# Releasing Cohorts
+
+This document explains what do once your [Pull Request](https://www.atlassian.com/git/tutorials/making-a-pull-request/) has been reviewed and all final changes applied. Now you're ready merge your branch into master and release it to the world:
+
+1. Assign a version to the release you are preparing. `cohorts` uses [versioneer](https://github.com/warner/python-versioneer), so rather
+than explicitly editing a `__version__` variable in the code you must instead tag the branch with your new version number (e.g. `git tag 1.2.3`).
+
+Warning: whether you should do `git tag v1.2.3` vs. `git tag 1.2.3` depends on versioneer settings, which can be found in `setup.cfg` and `_version.py` (because `_version.py` was generated from `setup.cfg` configuration, the two should match.) In `cohorts`, `git tag 1.2.3` (without the `v`) is what's expected.
```

Comment:
this seems nice for `hammerlab` release, but since we are doing  no `v` here mention that?